configuration/redis/metadata.yaml

```
@@ -20,27 +20,93 @@ metadata:
     description: Password for the Redis host
     example:  "KeFg23!"
     type: string
+  - name: redisUsername
+    required: false
+    description: Username for Redis host. Defaults to empty. Make sure your redis server version is 6 or above, and have created acl rule correctly.
+    example: "default"
+    type: string
```

I think it would be less confusing to state the default and user something as an example than "default".

```suggestion
  - name: redisUsername
    required: false
    description: Username for Redis host. Make sure your Redis server version is 6 or above, and have created ACLrule correctly.
    example: "myUsername"
    default: ""
    type: string
```

---

If it's not required then there is no default.